js/interact.js

```diff
@@ -195,15 +195,44 @@ function uiInteractLegend (metricQInstance, evtObj) {
     }
   }
   const timeString = posDate.toLocaleString()
-  myCtx.fillText(timeString, curPosOnCanvas[0] + 10, 40 - 20)
+  const offsetTop = 30
+  const verticalDiff = 20
+  const borderPadding = 10
+  const offsetMid = 10
+
+  myCtx.textBaseline = 'middle'
```

Comment:
We should set this within the two draw functions - otherwise those functions are dependent / make assumptions on the context in which they are executed. Even though in this case it's a slight redundancy.